Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```
@@ -559,6 +559,16 @@
       steps: 5
       zeroVisible: true
     - type: Appearance
+    - type: ExtendDescription
+      descriptionList:
+      - description: "civiliandisabler-extenddescription-security"
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 8 but found 6

(indentation)